docs/solutions/dr-pgbackrest-setup.md

```diff
@@ -0,0 +1,481 @@
+        /usr/bin/ssh-copy-id: INFO: Source of key(s) to be installed: "/root/.ssh/id_rsa.pub"
+        /usr/bin/ssh-copy-id: INFO: attempting to log in with the new key(s), to filter out any that are already installed
+        /usr/bin/ssh-copy-id: INFO: 1 key(s) remain to be installed -- if you are prompted now it is to install the new keys
+        postgres@pg-repo's password: 
+
+        Number of key(s) added: 1
+
+        
+        Now try logging into the machine, with:   "ssh 'postgres@pg-repo'"
+        and check to make sure that only the key(s) you wanted were added.
+        ```
+
+6. To verify everything has worked as expected, run the following command from the `pg-primary` node. 
+
+    ```sh
+    ssh postgres@pg-repo
+    ```
+
+    You should be able to connect to the `pg-repo` terminal without a password.
+
+7. Repeat the SSH connection from `pg-repo` to `pg-primary` to ensure that passwordless SSH is working. 
+8. Set up bidirectional passwordless SSH between `pg-repo` and `pg-secondary` using the same method. This will allow `pg-repo` to recover the backups to `pg-secondary`. 
+
+### Install Percona Distribution for PostgreSQL
+
+Install Percona Distribution for PostgreSQL in the primary and the secondary nodes from Percona repository. 
+
+1. [Install `percona-release`](https://www.percona.com/doc/percona-repo-config/installing.html).
+2. Enable the repository:
+
+    ```sh
+    sudo percona-release setup ppg11
+    ```
+
+3. Install Percona Distribution for PostgreSQL packages
+
+    - On Debian and Ubuntu:
+
+       ```sh
+       sudo apt install percona-postgresql-11 -y
+       ```
+   
+    - On RedHat Enterprise Linux and derivatives: 
+
+       ```sh
+       sudo yum install percona-postgresql11-server
+       ```
+
+### Configure PostgreSQL on the primary node for continuous backup
+
+At this step, configure the PostgreSQL instance on the `pg-primary` node for continuous archiving of the WAL files. 
+
+!!! note
+
+       On Debian and Ubuntu, the path to the configuration file is `/etc/postgresql/11/main/postgresql.conf`.
+
+       On RHEL and CentOS, the path to the configuration file is `/var/lib/pgsql/11/data/`.
+
+
+1. Edit the `postgresql.conf` configuration file to include the following changes:
+
+     ```
+     archive_command = 'pgbackrest --stanza=prod_backup archive-push %p'
+     archive_mode = on
+     listen_addresses = '*'
+     log_line_prefix = ''
+     max_wal_senders = 3
+     wal_level = replica
+     ```
+
+2. Once the changes are saved, restart PostgreSQL.
+
+    ```bash
+    sudo systemctl restart postgresql
+    ```
+
+### Install pgBackRest
+
+Install `pgBackRest` in all three instances from Percona repository. Use the following command:
+
+* On Debian / Ubuntu:
+  
+   ``` bash
+   sudo apt-get install percona-pgbackrest
+   ```
+
+* On RHEL / CentOS:
+
+   ``` bash
+   sudo yum install percona-pgbackrest
+   ```
+
+### Create the `pgBackRest` configuration file
```

Comment:
This section is asking the user to perform far too many operations that should really be handled by our packages.